TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0"
+          "version": "1.0.0-*"
```

Comment:
Try putting `"version": "1.0.0-servicing-*"` for the version.  The test failure makes it look like it is binding against the "1.0.0" version, and you want to break that.